src/applications/vaos/tests/project-cheetah/components/ContactInfoPage.unit.spec.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import { expect } from 'chai';
+import userEvent from '@testing-library/user-event';
+
+import ContactInfoPage from '../../../project-cheetah/components/ContactInfoPage';
+import { createTestStore, renderWithStoreAndRouter } from '../../mocks/setup';
+import { FETCH_STATUS } from '../../../utils/constants';
+import { cleanup } from 'axe-core';
+
+describe('VAOS <ContactInfoPage>', () => {
+  const initialState = {
+    projectCheetah: {
+      newBooking: {
+        data: {},
+        pages: [],
+        previousPages: [],
+      },
+    },
+  };
+
+  it('should submit with valid data', async () => {
+    const store = createTestStore(initialState);
+
+    let screen = renderWithStoreAndRouter(
+      <Route component={ContactInfoPage} />,
```

Comment:
You shouldn't need to wrap these in `Route` components. I think we still have some old tests doing that, but I don't believe it's necessary.